Sources/DeepLearning/Operators.swift

```diff
@@ -26,3 +26,326 @@ public func round<Scalar: BinaryFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Sc
 public func identity<Scalar>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
     return x
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Normalization
+//===------------------------------------------------------------------------------------------===//
+
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    // TODO: Verify that these calculations are correct.
+    @inlinable
+    func _vjpBatchNormalized(
+        alongAxis axis: Int32,
+        offset: Tensor,
+        scale: Tensor,
+        epsilon: Scalar
+        ) -> (Tensor, (Tensor) -> (Tensor, Tensor, Tensor)) {
```

Comment:
This is incorrectly indented. This should align with `func`.

Reply:
Fixed